changelog/fragments/component-config-changes.yaml

```diff
@@ -0,0 +1,10 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      This PR bumped Kubebuilder to a newer version and it got the recent PR changes for the `--component-config` flag.
+
+      If this flag is passed through the command line as an argument then only the `controller_manager_config.yaml` file will be scaffolded in the manifest. 
+      If it is not passed then it will not scaffold out this file. But, in this case, it will add `--leader-elect` to the files to the `manager.yaml` file
```

Comment:
What adds `--leader-elect`? All the plugins? some? Is that relevant to this portion?

Reply:
removed.